packages/react-vanilla-components/src/components/Panel.tsx

```diff
@@ -43,7 +41,9 @@ const Panel = function (props: PROPS_PANEL) {
         tooltip={props.tooltip}
         description={props.description}
       />
-      {renderChildren(props, context.mappings, context.modelId, handlers)}
+        <div className={props?.gridClassNames}>
```

Comment:
Add undefined check otherwise undefined class name will be added
props.gridClassNames || ''

Reply:
Added!